src/pybind/cephfs.py

```diff
@@ -317,6 +359,14 @@ def mkdirs(self, path, mode):
         if ret < 0:
             raise make_ex(ret, "error in mkdirs '%s'" % path)
 
+    def rmdir(self, path):
+        self.require_state("mounted")
+        if not isinstance(path, str):
```

Comment:
check for "basestring" rather than "str", to avoid falsely rejecting unicode string objects.

Actually, I have no idea what happens when you pass unicode into libcephfs itself... I expect we probably just accept it all and give you back dentries in the same encoding they were created.  Maybe the python layer should force strings into UTF-8 encoding before passing through to libcephfs?

@liewegas any thoughts on the unicode question?